input/data/animal_health/example_animal_health.json

```
@@ -0,0 +1,29 @@
+{
+    "disease1": {
+        "incidence": {
+            "baseline_incidence_rate": 1.8
+        },
+        "immediate_effects": {
+            "effect_1": {
+                "name": "milk_production_decrease",
+                "percent": 5 
+            },
+            "effect_2": {
+                "name": "miscarriage_likelihood_increase",
+                "percent": 3.5
+            }
```

```suggestion
            "milk_production_percentage_decrease": 5 ,
            "miscarriage_likelihood_percentage_increase": 3.5
```
I'm not sure what other data fields might need to be included for each effect, if they are not just limited to name and percent then my suggestion probably wouldn't be very helpful.

---

Yea, I don't know either. I might leave it this way for now just because I think there are a variety of effects and I'm not sure the flexibility we'll need/want for reading/implementing them.